framework/math/quadratures/angular/angular_quadrature.h

```diff
@@ -1,9 +1,8 @@
 #pragma once
 
+#include "framework/mesh/mesh_vector.h"
 #include <vector>
 
-#include "framework/mesh/mesh.h"
-
 namespace opensn
 {
 struct QuadraturePointPhiTheta;
```

Comment:
This struct name should change. But, maybe in a subsequent PR, the whole thing may be gone

Reply:
Yes, I suspect it will go away completely. I didn't touch much in the way of the internals for the angular quadrature sets. I'll do that in a future PR.